.snyk

```diff
@@ -34,4 +34,12 @@ ignore:
      - '*':
        reason: No direct upgrade or patch available
        expires: '2025-01-16T00:00:00.000Z'
+  SNYK-JS-PATHTOREGEXP-8482416:
+     - '*':
+       reason: No direct upgrade or patch available
+       expires: '2025-01-16T00:00:00.000Z'
+  SNYK-JS-AXIOS-6671926:
+     - '*':
+       reason: No direct upgrade or patch available
+       expires: '2025-01-16T00:00:00.000Z'
```

Comment:
These are patch updates - you could update them in the yarn.lock rather than ignoring them in the .snyk file

Reply:
Hi @Rhodine-orleans-lindsay thanks for the comments, I update the yarn.lock file.